### PR TITLE
Graph Node

### PR DESCRIPTION
**Pull request by unxnn**

### CODE REVIEW

File: charts/graph-node/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  - https://github.com/graphprotocol/graph-node
+
+dependencies:
+  - name: postgresql
+    version: 10.13.15
+    repository: https://charts.bitnami.com/bitnami/
+    condition: postgresql.enabled
+  - name: ipfs
+    version: 1.0.1
+    repository: "file://../ipfs"
```
Comment (tsudmi) [inline charts/graph-node/Chart.yaml:41]: I suggest we use the remote repo here

File: charts/graph-node/values.yaml [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+    indexStatus: 8030
+    # Port for Prometheus Exporter
+    metrics: 8040
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
```
Comment (tsudmi) [inline charts/graph-node/values.yaml:238]: Is it commented out in purpose?

File: charts/graph-node/values.yaml [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+
+## Configure resource requests and limits.
+## http://kubernetes.io/docs/user-guide/compute-resources/
+##
+resources: { }
```
Comment (tsudmi) [inline charts/graph-node/values.yaml:243]:
```suggestion
resources: {}
```

File: charts/graph-node/Chart.yaml (hunk 1 above)
Comment (unxnn) [reply charts/graph-node/Chart.yaml:41]: I will change when the charts are deployed and I can specify our repository, otherwise the pipeline will crash

**Review by tsudmi — APPROVED** [review]

**Pull request merged**